HARK/ConsumptionSaving/ConsGenIncProcessModel.py

```diff
@@ -1297,13 +1283,13 @@ class PersistentShockConsumerType(GenIncProcessConsumerType):
         pLvl is the permanent income level
 
         who_dies is the array of which agents died
-    aNrmInitMean: float
+    kLogInitMean: float
```

Comment:
The parameter names in the docstring were changed (e.g., `aNrmInitMean` → `kLogInitMean`). Please verify these match the actual parameter names used in the code.

Reply:
Yes, this is docstring cleanup from a prior PR. I happened to catch it while doing this one.